src/contexts/Validators/index.tsx

```diff
@@ -505,30 +505,33 @@ export const ValidatorsProvider = ({
           _validator = _validator.toHuman();
           let others = _validator.others ?? [];
 
-          // account for yourself being an additional nominator
+          // account for yourself being an additional nominator.
           const totalNominations = others.length + 1;
 
+          // reformat others.value properties from string to BigNumber.
+          others = others.map((other: AnyApi) => ({
+            ...other,
+            value: new BigNumber(rmCommas(other.value)),
+          }));
+
           // sort others lowest first.
-          others = others.sort((a: AnyApi, b: AnyApi) => {
-            const x = new BigNumber(rmCommas(a.value));
-            const y = new BigNumber(rmCommas(b.value));
-            return x.minus(y);
-          });
+          others = others.sort((a: AnyApi, b: AnyApi) =>
+            a.value.minus(b.value)
+          );
 
           // get the lowest reward stake of the validator, which is
           // the largest index - `maxNominatorRewardedPerValidator`,
           // or the first index if does not exist.
-          const lowestRewardIndex = Math.max(
-            others.length - maxNominatorRewardedPerValidator,
+          const lowestRewardIndex = BigNumber.max(
```

Comment:
I actually think this index should just be a `number`, we are just using it as an array index right now, which is a `number` anyway.